ds-prep-python-variables/main.py

```diff
@@ -0,0 +1,33 @@
+first_name = 'Tiffany'
+last_name = "Lin"
+full_name = (first_name + " " + last_name)
+print(full_name)
+height_in_inches = 62
+print ("height_in_inches:", height_in_inches, type(height_in_inches))
+height_in_inches_float = float(height_in_inches)
+print ("height_in_inches_float:", height_in_inches_float , type (height_in_inches_float))
+height_in_meters = 0.0254 * height_in_inches_float
+print ("height_in_meters:", height_in_meters)
+
+eats_plants = True
+eats_animals = False
+is_animal = eats_animals or eats_plants
+print (" is_animal: ",(eats_plants or eats_animals))
```

Comment:
Rather than re-calculating the value, just directly use `is_animal` in your `print`.